yarn.lock

```diff
@@ -2,6 +2,13 @@
 # yarn lockfile v1
```

Comment:
Is this a good opportunity to switch to v2? Perhaps now, but in a separate PR?

What would be the recommendation here to work locally? 
Have both, Yarn v2+ for Grafana and Yarn v1 for this project?

Thanks!

Reply:
Not sure what would be best here, I'll investigate switching to V2 on a separate PR.

Reply:
Great, thanks!

PS: Honestly, if there's any easy way to have/use both with no big pains, I'd be happy to keep it as-is, although it's something we'll need to revisit anyway at some time, yeah.